check-approved-requests.py

```diff
@@ -203,6 +210,15 @@ def check_requests(request_type, auth_file, worksheet_key):
             processed_rows.append(idx)
             if args.log:
                 log_request(args.log, timestamp, request_info['user_email'])
+
+        elif (row[0] == '') and (datetime.now() >= dateparser.parse(row[2]) + timedelta(hours=24)):
+            # process rows that has been waiting for approval for more than 24 hours
```

Comment:
maybe "send a reminder about rows..." would be a bit more descriptive here, since you are editing anyway